### PR TITLE
fnt 1.4.1 (new formula)

### PR DESCRIPTION
**Pull request by zhangcheng**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by alebcay — COMMENTED** [review]
The file `fnt.rb` should be created inside the `Formula` folder, not at the root of the repository (this is causing the CI failure).

File: Formula/fnt.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  depends_on "lcdf-typetools"
+
+  def install
+    bin.install "fnt"
+    man1.install "fnt.1"
+    zsh_completion.install "completions/_fnt"
+  end
+
+  test do
+    system "#{bin}/fnt", "info"
```
Comment (nandahkrishna) [inline Formula/fnt.rb:18]:
```suggestion
    assert_match "Available Fonts", shell_output("#{bin}/fnt info")
```

**Comment by zhangcheng** [comment]
Thanks @alebcay and @nandahkrishna , updated per your comments.

**Review by nandahkrishna — APPROVED** [review]
Thanks, @zhangcheng! 🍻

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**